docker/container.go

```diff
@@ -343,6 +358,10 @@ func (c *Container) createContainer(imageName, oldContainer string) (*types.Cont
 	configWrapper.Config.Labels[SERVICE.Str()] = c.service.name
 	configWrapper.Config.Labels[PROJECT.Str()] = c.service.context.Project.Name
 	configWrapper.Config.Labels[HASH.Str()] = c.getHash()
+	// libcompose run command not yet supported, so always "False"
+	configWrapper.Config.Labels[ONEOFF.Str()] = "False"
+	configWrapper.Config.Labels[NUMBER.Str()] = getContainerNumber(c.name)
```

Comment:
I would be better to pass in the number instead of having to parse it from the name.